src/apps/Altinn.AccessManagement/src/Altinn.AccessManagement.Core/Errors/ValidationErrors.cs

```diff
@@ -76,4 +76,130 @@ private static readonly ValidationErrorDescriptorFactory _factory
     /// </summary>
     public static ValidationErrorDescriptor AssignmentIsActiveInOneOrMoreDelegations { get; }
         = _factory.Create(10, $"Assignment is active in one or more delegations.");
+
+    /// <summary>
+    /// Gets a validation error descriptor
+    /// </summary>
+    public static ValidationErrorDescriptor NotAuthorizedForConsentRequest { get; }
+        = _factory.Create(19, "Not Authorized for ");
+
+    /// <summary>
+    /// Gets a validation error descriptor
+    /// </summary>
+    public static ValidationErrorDescriptor InvalidPersonIdentifier { get; }
+        = _factory.Create(20, "Invalid person identifier. No person with this identifier.");
```

Comment:
I don't think saying "No person with this identifier." is reasonagble. You should not get a validation error if you provided a valid person identifier but the person does not exist.

Reply:
changed to problem